web_widget_dropdown_dynamic/static/tests/web_widget_dropdown_dynamic_tests.esm.js

```diff
@@ -1,8 +1,6 @@
-/** @odoo-module **/
-
+/* eslint-disable no-undef */
 import {editInput, getFixture} from "@web/../tests/helpers/utils";
 import {makeView, setupViewRegistries} from "@web/../tests/views/helpers";
-import {QUnit} from "@web/static/lib/qunit/qunit";
```

Comment:
Isn't it `QUnit` used a bit below? Could you explain the rationale behind the patch?